src/ProfileEvent.h

```diff
@@ -321,8 +324,22 @@ class ProfileEventSingleton {
             return running;
         }
 
+        void increaseInterval() {
+            if (t < 60000) {
```

Comment:
Add a comment to explain the limit. e.g.
// Don't increase time interval past 60 seconds